deploy/local_single_user_deploy.sh

```diff
@@ -465,34 +389,50 @@ usage() {
+    echo "  -l|--install-dir <path>          Define installation directory (default: /home/$USER/chronolog/BUILD_TYPE) [Modes: Build]"
     echo ""
     echo "Deployment Options:"
-    echo "  -k|--keepers        Set the total number of keeper processes. They will be assigned iteratively to the recording groups [Modes: Start]"
-    echo "  -r|--record-groups  Set the number of recording groups or grapher processes [Modes: Start]"
+    echo "  -k|--keepers <number>            Set the total number of keeper processes. They will be assigned iteratively to the recording groups (default: 1)[Modes: Start]"
+    echo "  -r|--record-groups <number>      Set the number of recording groups or grapher processes (default: 1)[Modes: Start]"
     echo ""
     echo "Directory Settings:"
-    echo "  -w|--work-dir       WORK_DIR Set the working directory (Mandatory) [Modes: Install, Start, Stop, Clean]"
-    echo "  -m|--monitor-dir    MONITOR_DIR (default: work_dir/monitor) [Modes: Start]"
-    echo "  -u|--output-dir     OUTPUT_DIR (default: work_dir/output) [Modes: Start]"
+    echo "  -w|--work-dir <path>             Set the working directory (Mandatory) [Modes: Install, Start, Stop, Clean]"
+    echo "  -m|--monitor-dir <path>          Set the monitor directory (default: work_dir/monitor) [Modes: Start]"
+    echo "  -u|--output-dir <path>           Set the output directory (default: work_dir/output) [Modes: Start]"
     echo ""
     echo "Binary Paths:"
-    echo "  -v|--visor-bin      VISOR_BIN (default: work_dir/bin/chronovisor_server) [Modes: Start]"
-    echo "  -g|--grapher-bin    GRAPHER_BIN (default: work_dir/bin/chrono_grapher) [Modes: Start]"
-    echo "  -p|--keeper-bin     KEEPER_BIN (default: work_dir/bin/chrono_keeper) [Modes: Start]"
-    echo "  -a|--player-bin     PLAYER_BIN (default: work_dir/bin/chrono_player) [Modes: Start]"
+    echo "  -v|--visor-bin <path>            Path to the ChronoVisor binary (default: work_dir/bin/chronovisor_server) [Modes: Start]"
+    echo "  -g|--grapher-bin <path>          Path to the ChronoGrapher binary (default: work_dir/bin/chrono_grapher) [Modes: Start]"
+    echo "  -p|--keeper-bin <path>           Path to the ChronoKeeper binary (default: work_dir/bin/chrono_keeper) [Modes: Start]"
+    echo "  -a|--player-bin <path>           Path to the ChronoPlayer binary (default: work_dir/bin/chrono_player) [Modes: Start]"
     echo ""
     echo "Configuration Settings:"
-    echo "  -f|--conf-file      CONF_FILE Path to the configuration file (default: work_dir/conf/default_conf.json) [Modes: Start]"
+    echo "  -f|--conf-file <path>            Path to the configuration file (default: work_dir/conf/default_conf.json) [Modes: Start]"
     echo ""
     echo "Examples:"
-    echo "  $0 --build --build-type Debug"
-    echo "  $0 --start --work-dir /path/to/workdir --conf-file /path/to/config.json"
+    echo "  1) Builds ChronoLog in Debug mode:"
```

Comment:
I suggest we add something to the `Examples:` line to distinguish `install-dir` and `work-dir` in the examples. Like `Examples (assume installing a Debug build to ~/chronolog_install`. Then we can give specified paths in the examples. For `install-dir` in Build mode, we say `~/chronolog_install`. For `work-dir` in the other examples, we say `~/chronolog_install/Debug`. With this, it's easier for the user to understand the difference between those two directories.